### ISSUE TITLE
Using with GOG version

### ISSUE THREAD

**Issue by mbajur** [exult/exult#23]
First of all - thank you for this amazing piece of software and sorry for such a lame question. I'm just not sure if that's even possible so i don't want to waste my time if it's not.

I've just bought an Ultima VII collection on GOG.com which is probably the most common go-to place for buying Ultima VII right now. However, it seems that they're packing the game in some sort of non-standard way which gives me some troubles while trying to follow instructions in the README.

First thing is - i'm on a Mac, i have two `.app` files for both Black Gate and SI: `Ultima VII™  - The Black Gate + The Forge of Virtue.app` and `Ultima VII™  - Serpent Isle + The Silver Seed.app`. What i've found in the README is that i should put these `.app` files somewhere in `~/Library/Application Support/Exult` but i'm not sure where to put them (in `blackgate` and `serpentisle` maybe?) nor how to rename the `.app` files (as their original names are pretty complicated, i'm almost sure it can cause some troubles). No matter what i try, Exult gives me an error that it can't find exult data files.

I first want to try to do that without touching the exult.cfg file as it's not an issue for me to move the app files here and there. If that's not possible, i'm gonna try the exult.cfg approach.

Thanks!

***

Not related, probably it deserves a separate issue: i've seen you guys migrated from sourceforge to github but your website is still on sourceforge. You need some help with migrating the website to github as well? It can be nicely integrated with that repo, can be auto-deplyed under https://exult.github.io domain when readme changes etc, pretty convenient :)

**Comment by DominusExult** [comment]
Hi, using the GoG version is a great way to play the Ultimas. Unfortunately they package the games inside the app in some strange way.
I did write a guide on how to best make use of it but didn't update it yet after GoG changed the details for the nth time ;)
But it's a good way to start: read http://exult.sourceforge.net/docs.php#gog_mac
The "sudo mkdir..." commands are still valid but in the "sudo cp..." commands you need to exchange /Volumes/Ultima 7/Ultima 7 The Complete Edition.app with /Applications/name of the BG.app and in the second command /Apllicstions/name of the SI.app (but the remainder of the path should be the same but I can't check right now).
Let me know if you get by with this rough guide or if you need more details.

Homepage: we are still staying on SF on that mainly because our simple forum uses php and AFAIK that's the one weakness of the github webpages.

**Comment by DominusExult** [comment]
And please pay attention to the difference of /Library/... and ~/Library/...
The first is in the root of your system and the other in your home folder.
Exult looks for the games in /Library/Application Support/Exult but saves to ~/Library/Application Support/Exult

**Comment by mbajur** [comment]
Yes! I have no idea why that tutorial you wrote on exult page didn't showed up in google results but it did the trick. I just had a bit different files setup so i needed to modify the commands a bit. If anyone would need that in the future, here is what i did:

```
sudo mkdir -p "/Library/Application Support/Exult/blackgate"
sudo mkdir -p "/Library/Application Support/Exult/serpentisle"

sudo cp -p -R "/Applications/Ultima/Ultima VII™  - The Black Gate + The Forge of Virtue.app/Contents/Resources/Ultima 7 The Complete Edition.boxer/C.harddisk/U7/ULTIMA7/STATIC" "/Library/Application Support/Exult/blackgate"

sudo cp -p -R "/Applications/Ultima/Ultima VII™  - Serpent Isle + The Silver Seed.app/Contents/Resources/Ultima 7 The Complete Edition.boxer/C.harddisk/U7/SERPENT/STATIC" "/Library/Application Support/Exult/serpentisle"
```

Thank you :)

Homepage: well you could keep the forum on sourceforge and move all the rest to github. Anyway, if you need any help, feel free to ping me anytime.

**Comment by rofl0r** [comment]
> so i don't want to waste my time if it's not

you should be more concerned about wasting **other people**'s time

**Comment by mbajur** [comment]
seriously @rofl0r ? Have i wasted anybody's time? I apologized for my question in it's first sentence. I'm contributing to a lot of open source projects and believe it or not, i do know how to use google, creating an issue like this is an extreme and it means i was not able to find what i was looking for. I've read a lot of forum posts on gog forums (that was the nly place found on google) and they were not handy at all. I tried following a readme - without any luck (in cause of how GOG packs their games, not my nor exult fault).

After ~2h of trial and error, i decided to create an issue for a thing that seems to be easy but it's not (GOG's fault, not yours). I was polite and some friendly person (@DominusExult, thank you, you're the best) decided to help me with this thing, which turned out to be not as obvious as i could suppose. The only weak point of this whole operation was to not be able to find a GOG mention in the readme when googling, but that's how google works. There is no separate website with GOG guide but one big html file with #anchor links - google is not good at that + gog.com is probably way better at SEO.

Anyway, i decided to ask, i received an answer, i modified it to match my specific use case and, finally, i shared my solution if anyone in the future will face the same difficulties. If i would not be concerned about **other people**'s time, i would not do it all, especially the last part.

If in your opinion, it's better to drop using something because it's impossible to find a solution rather than friendlily ask for a help and share the working solution for others at the end of the process - i feel sorry for you and for your coworkers. 

Cheers!
